### PR TITLE
Several minor imrpovements

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: _posts/getting-started/2018-01-01-installation-guide.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Since Windows machines don't use bash, we recommend that you install git and inc
 To update the CLI to the latest supported version, run the following command:
 
 ```bash
-npm update -g decentraland
+npm install -g decentraland
```
Comment (matiasbargas) [inline _posts/getting-started/2018-01-01-installation-guide.md:65]: I would recommend to remove it first when update the cli

Comment (nearnshaw) [reply _posts/getting-started/2018-01-01-installation-guide.md:65]: I just pushed another commit to include that

File: _posts/blockchain-interactions/2018-02-01-display-a-certified-nft.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ set_order: 4
 
 You can display an NFT (Non-Fungible Token) that you own in your Decentraland scenes. A glowing badge will appear above the NFT to certify that you indeed own it.
 
-![](/images/media/verified-nft.png)
+<img src="/images/media/verified-nft.png" alt="nested entities" width="300"/>
```
Comment (eordano) [inline _posts/blockchain-interactions/2018-02-01-display-a-certified-nft.md:14]: How does this look like on mobile?

File: _posts/deploy/2018-01-01-deploy-to-now.md [hunk 3]
```diff
@@ -17,8 +17,8 @@ Once uploaded to Now, the only thing that others have to do in order to explore
 
 For example here are some scenes that are currently running in Now:
 
-- Block Dog
-- Humming birds
+- [Block Dog](https://blockdog-wtciaozdbo.now.sh/?position=0%2C0)
```
Comment (eordano) [inline _posts/deploy/2018-01-01-deploy-to-now.md]: take out the position?

**Review by eordano — APPROVED** [review]

**Comment by eordano** [comment]
Looks good to go, just some minor comments but not blocking.

**Review by matiasbargas — APPROVED** [review]

**Comment by adrabenche** [comment]
Mannually checked the unverified commits beongs to @nearnshaw.

Also set up to from now on signe the commits with the right key.

**Pull request merged**